src/logging.cpp

```diff
@@ -319,6 +319,18 @@ std::vector<LogCategory> BCLog::Logger::LogCategoriesList() const
     return ret;
 }
 
+/** Log severity levels that can be selected by the user. */
+static constexpr std::array<BCLog::Level, 2> LogLevelsList()
+{
+    return {BCLog::Level::Info, BCLog::Level::Debug};
+}
+
+std::string BCLog::Logger::LogLevelsString() const
+{
+    const auto& levels = LogLevelsList();
+    return Join(std::vector<BCLog::Level>{levels.begin(), levels.end()}, ", ", [this](BCLog::Level level) { return LogLevelToStr(level); });
```

Comment:
nit in 9c7507bf76e79da99766a69df939520ea0a125d1 (only if you retouch and rebase for other reasons):

Can drop std::vector copy after https://github.com/bitcoin/bitcoin/commit/c89fabff3d17f23098d70c925fbdc98aca79fec6

Reply:
Thanks, will touch up in the parent (leaving unresolved until it's done).